fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/BaseWhereAdapter.java

```diff
@@ -0,0 +1,1137 @@
+    }
+
+    /**
+     * Add < {ref} to the expression
+     * @param ref the column reference
+     * @return
+     */
+    public T lt(ColumnRef ref) {
+        predicateParser.lt();
+        predicateParser.column(ref.toString());
+        return getThis();
+    }
+
+    /**
+     * Add < {literalValue} to the expression
+     * @param literalValue
+     * @return
+     */
+    public T lt(long literalValue) {
+        predicateParser.lt();
+        predicateParser.literal(literalValue);
+        return getThis();
+    }
+
+    /**
+     * Add < {literalValue} to the expression
+     * @param literalValue
+     * @return
+     */
+    public T lt(int literalValue) {
+        predicateParser.lt();
+        predicateParser.literal((long)literalValue);
+        return getThis();
+    }
+
+    /**
+     * Add < {literalValue} to the expression
+     * @param literalValue
+     * @return
+     */
+    public T lt(String literalValue) {
+        predicateParser.lt();
+        predicateParser.literal(literalValue);
+        return getThis();
+    }
+
+    /**
+     * Add < {bindMarker} to the expression
+     * @param bindMarker
+     * @return
+     */
+    public T lt(BindMarkerNode bindMarker) {
+        predicateParser.lt();
+        predicateParser.bindMarker(bindMarker);
+        return getThis();
+    }
+
+    // ### LESS THAN EQUAL ###
+    /**
+     * Add <= to the expression
+     * @return
+     */
+    public T lte() {
+        predicateParser.lte();
+        return getThis();
+    }
+
+    /**
+     * Add <= ref to the expression
+     * @param ref
+     * @return
+     */
+    public T lte(ColumnRef ref) {
+        predicateParser.lte();
+        predicateParser.column(ref.toString());
+        return getThis();
+    }
+
+    /**
+     * Add <= {literalValue} to the expression
+     * @param literalValue
+     * @return
+     */
+    public T lte(long literalValue) {
+        predicateParser.lte();
+        predicateParser.literal(literalValue);
+        return getThis();
+    }
+
+    /**
+     * Add <= {literalValue} to the expression
+     * @param literalValue
+     * @return
+     */
+    public T lte(int literalValue) {
+        predicateParser.lte();
+        predicateParser.literal((long)literalValue);
+        return getThis();
+    }
+
+    /**
+     * Add <= {literalValue} to the expression
+     * @param literalValue
+     * @return
+     */
+    public T lte(String literalValue) {
+        predicateParser.lte();
+        predicateParser.literal(literalValue);
+        return getThis();
+    }
+
+    /**
+     * Add <= {bindMarker} to the expression
+     * @param bindMarker
+     * @return
+     */
+    public T lte(BindMarkerNode bindMarker) {
+        predicateParser.lte();
+        predicateParser.bindMarker(bindMarker);
+        return getThis();
+    }
+
+    /**
+     * Add a column expression or reference
+     * @param expression
+     * @return
+     */
+    public T col(String expression) {
```

Comment:
334 has similar method body and similar signature...

Reply:
expr(...) wasn't used so I've removed it.